sdk/python/kfp/components/pipeline_task.py

```diff
@@ -474,6 +475,28 @@ def my_pipeline():
             self._task_spec.dependent_tasks.append(task.name)
         return self
 
+    def ignore_upstream_failure(self) -> 'PipelineTask':
+        """If called, the pipeline task will run when any specified upstream
+        tasks complete, even if unsuccessful.
+
+        This method effectively turns the caller task into an exit task
+        if the caller task has upstream dependencies.
+        """
```

Comment:
I think we should probably add the same `Returns:` section found in other task methods to the docstring. WDYT?